packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -70,5 +70,9 @@ export const messages = {
   force_lightning_app_create_text: 'SFDX: Create Lightning App',
   force_lightning_component_create_text: 'SFDX: Create Lightning Component',
   force_lightning_event_create_text: 'SFDX: Create Lightning Event',
-  force_lightning_interface_create_text: 'SFDX: Create Lightning Interface'
+  force_lightning_interface_create_text: 'SFDX: Create Lightning Interface',
+  warning_prompt_file_override:
```

Comment:
@ruthemmanuelle Could you take a look at this message? The context is what to prompt when the user tries to create an Apex/Lightning/VF file that already exists. I made it 'file(s)' because in Lightning's case, multiple files will be overwritten.